api/envoy/api/v2/auth/cert.proto

```diff
@@ -391,6 +391,9 @@ message DownstreamTlsContext {
     // Config for fetching TLS session ticket keys via SDS API.
     SdsSecretConfig session_ticket_keys_sds_secret_config = 5;
   }
+
+  // If specified, session_timeout is used as a hint for tls tickets lifetime
```

Comment:
There isn't enough in this comment to understand what the field does.

Reply:
sure. will add more info.